niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/InstallPermissionTester.java

```diff
@@ -486,10 +498,19 @@ public void onAvailable(Network network) {
         // new install permissions for Q
         mPermissionTasks.put(REQUEST_PASSWORD_COMPLEXITY,
                 new PermissionTest(false, Build.VERSION_CODES.Q,
-                        () -> mDevicePolicyManager.getPasswordComplexity()));
+                        () -> {
+                            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
```

Comment:
Nit: the device SDK level check is not required since the `PermissionTest` already specifies a minSdkVersion of Q, so it will automatically be skipped on earlier SDK versions. Or was this check added because of a lint warning that doesn't recognize the minSdkVersion check at the `BasePermissionTester` level?

Reply:
It intend to suppress the lint error.
To suppress that without this syntax, user need to disable below lint settings on Android Studio.

Editor->Inspections->Android -> Lint -> Correctness -> Calling new methods on older versions.

I think it might confuses user. please let me know your thoughts.

Reply:
Sorry I missed this comment before I added similar comments in other code locations. I agree it is better to have these extra checks in place to suppress the lint errors; these extra conditionals will likely not be noticed, but the lint errors may surface and cause confusion. Thanks for letting me know about the lint errors!